src/tools/qt-gui/qml/KeyWindow.qml

```diff
@@ -116,13 +116,13 @@ BasicWindow {
 					metaValueField.placeholderText: valuePlaceHolder
 
 					metaNameField.onTextChanged:  {
-						if(metaName !== metaNameField.text){
+                        if(metaName.trim() !== metaNameField.text.trim()){
```

Comment:
I wanted to avoid that a white space causes an edit action, but was not aware that keynames with whitespaces are desired.